src/components/Buttons/RadioCheckbox.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component, PropTypes }        from 'react'
+import Radium                                 from 'radium'
+import svgRadioBtnBase                        from '../../assets/radioBtnBase.svg'
+import svgRadioBtnChecked                     from '../../assets/radioBtnChecked.svg'
+import svgCheckboxBase                        from '../../assets/checkboxBase.svg'
+import svgCheckboxChecked                     from '../../assets/checkboxChecked.svg'
+
+const BASE_STYLES = {
+  input: {
+    margin: '0 10px -7px 0',
+    backgroundSize: 'contain',
+    backgroundRepeat: 'no-repeat',
+    backgroundPosition: 'top left',
+    backgroundColor: 'transparent',
+    appearance: 'none',
+    '-moz-appearance': 'none',
+    '-webkit-appearance': 'none',
+    width: 22,
+    height: 22,
+  },
+}
+
+const STYLES = {
+  radioBase: {
+    ...BASE_STYLES.input,
+    backgroundImage: `url(${svgRadioBtnBase.url})`,
+    ':active': {
+      backgroundImage: `url(${svgRadioBtnBase.url})`,
+      border: '0 solid transparent'
+    }
+  },
+  radioChecked: {
+    backgroundImage: `url(${svgRadioBtnChecked.url})`,
+    ':active': {
+      backgroundImage: `url(${svgRadioBtnChecked.url})`,
+      border: '0 solid transparent'
+    }
+  },
+  checkboxBase: {
+    ...BASE_STYLES.input,
+    backgroundImage: `url(${svgCheckboxBase.url})`,
+    ':active': {
+      backgroundImage: `url(${svgCheckboxBase.url})`,
+      border: '0 solid transparent'
+    }
+  },
+  checkboxChecked: {
+    backgroundImage: `url(${svgCheckboxChecked.url})`,
+    ':active': {
+      backgroundImage: `url(${svgCheckboxChecked.url})`,
+      border: '0 solid transparent'
+    }
+  }
+}
+
+class RadioCheckbox extends Component {
```

Comment:
I'd be a fan of exporting multiple components, e.g. `Checkbox` and `Radio`, with `Radio` being a small wrapper around `Checkbox`. I like it mostly because the API feels more self-explanatory. Docs could be short for `Radio` as well, just a small example and maybe link back to the checkbox docs

Reply:
Agreed - that seems simpler and more obvious for users of the library

Reply:
It may feel like more work for us, but it will feel like a more clear and easier to use library from the outside. That's what we're aiming for :)